lib/omnibus/packagers/bff.rb

```diff
@@ -229,6 +232,9 @@ def create_bff_file
       FileSyncer.glob(File.join(staging_dir, 'tmp/*.bff')).each do |bff|
         copy_file(bff, File.join(Config.package_dir, create_bff_file_name))
       end
+
+      # chown back to original user's uid/gid so cleanup works correctly
+      shellout!("sudo chown -Rh #{original_uid}:#{original_gid} #{staging_dir}")
```

Comment:
I'm not sure it totally matters, if there is an exception we want to keep the directory intact in the state it was in when it failed, no?